test/ServerComparison.FunctionalTests/NtlmAuthenticationTest.cs

```diff
@@ -40,10 +41,15 @@ public class NtlmAuthenticationTests
                     EnvironmentName = "NtlmAuthentication", // Will pick the Start class named 'StartupNtlmAuthentication'
                     ServerConfigTemplateContent = Helpers.GetConfigContent(serverType, "NtlmAuthentication.config", nginxConfig: null),
                     SiteName = "NtlmAuthenticationTestSite", // This is configured in the NtlmAuthentication.config
-                    ApplicationType = applicationType,
-                    TargetFramework = "net452",
+                    TargetFramework = runtimeFlavor == RuntimeFlavor.Clr ? "net452" : "netcoreapp1.1",
+                    ApplicationType = applicationType
                 };
 
+                if(applicationType == ApplicationType.Standalone)
+                {
+                    deploymentParameters.AdditionalPublishParameters = " -r " + RuntimeEnvironment.GetRuntimeIdentifier();
```

Comment:
Consider making this a first class concept rather than manually adding the parameters in every test.